### PR TITLE
Fix fp-ts deprecation warnings in equality examples

### PR DESCRIPTION
**Pull request by adomokos**

### CODE REVIEW

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: We should not import from `lib` anymore, the following should work:

```suggestion
import { Eq } from "fp-ts/Eq";
```

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]:
```suggestion
import { Eq } from "fp-ts/Eq";
```

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: I'm surprised that we need to provide the generic value here, doesn't this work without `<string>`? (I haven't tried it)

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]:
```suggestion
import { Eq } from "fp-ts/Eq";
```

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: We can use https://gcanti.github.io/fp-ts/modules/Eq.ts.html#struct here.

File: docs/equality.md
Comment (grossbart) [inline docs/equality.md]: Does this also work without providing the generic variable? Same as further up, I would hope that it is not necessary anymore.

**Review by grossbart — CHANGES_REQUESTED** [review]
Thanks for updating this! I have added some small things that should be fixed before merging. If you find the time to do these, that would be great, if not I will try to fix it later.

File: docs/equality.md
Comment (adomokos) [reply docs/equality.md]: I'll check.

File: docs/equality.md
Comment (adomokos) [reply docs/equality.md]: Fixed it.

File: docs/equality.md
Comment (adomokos) [reply docs/equality.md]: Fixed it.

File: docs/equality.md
Comment (adomokos) [reply docs/equality.md]: Removed.

File: docs/equality.md
Comment (adomokos) [reply docs/equality.md]: Fixed it.

File: docs/equality.md
Comment (adomokos) [reply docs/equality.md]: Removed.

**Comment by adomokos** [comment]
@grossbart, thanks for the improvement suggestions. Please take a look again, hope this is polished enough to merge it.

**Review by grossbart — APPROVED** [review]
Thanks for the quick polishing and your initiative, will merge right now :)

**Pull request merged**
